packages/blade/src/components/Typography/BaseText/getBaseTextStyles.ts

```diff
@@ -32,6 +32,7 @@ const getBaseTextStyles = ({
         'line-clamp': `${numberOfLines}`,
         '-webkit-line-clamp': `${numberOfLines}`,
         '-webkit-box-orient': 'vertical',
+        wordBreak: 'break-word',
```

Comment:
We should not put wordBreak by default on BaseText, this is adding this property to every single blade component since most components use BaseText. 

Instead we can expose a new prop for `wordBreak` and also expose it in the Text component. 
And then add wordBreak="break-all" to [FormLabel](https://github.com/razorpay/blade/blob/360abe731379741de71c051422cf58dc73b8f70c/packages/blade/src/components/Form/FormLabel.tsx#L108-L117)

Reply:
yes